contracts/modules/STO/USDTieredSTOStorage.sol

```diff
@@ -66,17 +75,21 @@ contract USDTieredSTOStorage {
     // Amount in fund raise type invested by each investor
     mapping (address => mapping (uint8 => uint256)) public investorInvested;
 
-    // List of accredited investors
-    mapping (address => bool) public accredited;
+    // Accredited & non-accredited investor data
+    mapping (address => Investor) public investors;
 
     // List of active stable coin addresses
     mapping (address => bool) public usdTokenEnabled;
 
+    // List of all addresses that have been added as accredited or non-accredited without
+    // the default limit
+    address[] public investorsList;
+
     // Default limit in USD for non-accredited investors multiplied by 10**18
     uint256 public nonAccreditedLimitUSD;
 
     // Overrides for default limit in USD for non-accredited investors multiplied by 10**18
```

Comment:
This commented code can also be removed

Reply:
@F-OBrien which code? I think only the comments are commented here.

Reply:
// Overrides for default limit in USD for non-accredited investors multiplied by 10**18
    /* mapping (address => uint256) public nonAccreditedLimitUSDOverride; */

Reply:
sorry looks like I had put my comment against the line above this

Reply:
Thanks - done.